modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/service/test/DLCheckInCheckOutTest.java

```diff
@@ -572,6 +633,25 @@ protected FileEntry updateFileEntry(long fileEntryId, String fileName)
 			content.length(), null, null, null, _serviceContext);
 	}
 
+	private void _setToOldStoreFile() throws Exception {
+		DLFileEntry dlFileEntry = DLFileEntryLocalServiceUtil.getDLFileEntry(
+			_fileEntry.getFileEntryId());
+
+		DLFileVersion dlFileVersion = dlFileEntry.getFileVersion();
+
+		DLStoreUtil.updateFile(
+			DLStoreRequest.builder(
+				dlFileEntry.getCompanyId(), dlFileEntry.getDataRepositoryId(),
+				dlFileEntry.getName()
+			).versionLabel(
+				dlFileVersion.getVersion()
+			).build(),
+			new UnsyncByteArrayInputStream(_TEST_CONTENT.getBytes()));
+		DLStoreUtil.deleteFile(
+			dlFileEntry.getCompanyId(), dlFileEntry.getDataRepositoryId(),
+			dlFileEntry.getName(), dlFileVersion.getStoreFileName());
+	}
+
```

Comment:
Can you rename this to something like `_renameDLStoreFile`.

Reply:
<img src="https://media0.giphy.com/media/tIeCLkB8geYtW/giphy.gif"/>